nntrainer/layers/meson.build

```diff
@@ -9,6 +9,7 @@ layer_sources = [
   'subtract_layer.cpp',
   'multiply_layer.cpp',
   'divide_layer.cpp',
+  'pow_layer.cpp',
```

Comment:
This is a simple suggestion (not directly related to this PR though).
What about using prefix like 'op_' to operation layers?
It might reduce the confusion on the layer types (e.g., add_layer & addition_layer).

Reply:
That sounds good idea. I'll reflect it in later pr. thanks!